src/loader.ts

```diff
@@ -244,6 +245,18 @@ module.exports = async function loader(
         }
 
         debug(`resolved request to: ${style.absoluteFilePath}`);
+
+        // replace composes first bc we need need to use a different identifier
+        localStyle.value = replaceComposes(localStyle.value, (match) =>
+          match
+            .replace(source, `~${style.absoluteFilePath}`)
+            .replace(
+              imported,
+              style.type === 'stylesheet' ? memberProperty : 'cls2',
+            ),
+        );
```

Comment:
**question:** is this code path tested? seems like perhaps not?

Reply:
it's tested inso far as there are snapshots that capture it. But obviously not tested in such a way that i noticed it was broken. It wasn't until i added a manual test elsewhere it was clear it was broken. Maybe we should add some percy tests